.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+# Welcome to Cryostat! 👋
```

Comment:
Also here and similar places do we update to cryostat3 or just leave it like that for now?

Reply:
I think we'll just leave it like that. Ideally, this repo continues on to be Cryostat 3, and 4 and 5 and however many major versions there may  be in the future. 3.0 is the successor to 2.x and I don't want to have to do a major rebase/reimplementation like the split between 1.x/2.x and 3.0 again :-)

I think at some point this repo will get renamed to simply `cryostat`, and the old repo will get renamed in some way that reflects that it's the original codebase but replaced by this one.